hal-base/hal/functions.py

```diff
@@ -494,14 +484,14 @@ def cleanNotifier(notifier):
 #############################################################################
 # PDP
 #############################################################################
-getPDPTemperature = _STATUSFUNC("getPDPTemperature", C.c_double)
```

Comment:
An Issue I am running into is that on some functions, like these PDP functions, the "status" argument is not the final argument in the c++ API, like we expect. I suspect someone got lazy when they added arguments to these, and didn't rearrange them to match the rest of the API. See https://github.com/robotpy/allwpilib/blob/master/hal/include/HAL/PDP.hpp